.github/workflows/release.yml

```diff
@@ -24,19 +27,25 @@ jobs:
         with:
           node-version-file: '.nvmrc'
 
-      - name: Get yarn cache
-        id: yarn-cache
-        run: echo "::set-output name=dir::$(yarn config get cacheFolder)"
+      - name: Install pnpm
+        uses: pnpm/action-setup@v2.2.3
+        id: pnpm-install
+        with:
+          run_install: false
+
+      - name: Get pnpm store directory
```

Comment:
I think the cache-action might support this natively. https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#caching-packages-data

Reply:
Thanks, I'll try that